alien4cloud-core/src/main/java/alien4cloud/topology/DependencyConflictDTO.java

```diff
@@ -0,0 +1,20 @@
+package alien4cloud.topology;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * Represents a dependency conflict for a source CSAR in a topology.
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class DependencyConflictDTO {
+
+    private String source;
+    private String expected;
```

Comment:
/** Expected is the archiveName:archiveVersion that causes a dependency conflict. */
Should be renamed to dependency ? or expectedDependency ?